skr-webhook/resources.yaml

```diff
@@ -107,7 +95,7 @@ spec:
                   key: contractVersion
             - name: WEBHOOK_SIDE_CAR
               value: "true"
-          image: "europe-docker.pkg.dev/kyma-project/prod/runtime-watcher-skr:latest"
+          image: "europe-docker.pkg.dev/kyma-project/prod/runtime-watcher-skr:v20230824-f4459bad"
```

Comment:
is this intended?

Reply:
yes, i would rather like to pin it to a specific version, than use latest, because any merge to runtime-watcher main will trigger the post build and create a new latest. Just to make it deliberate to bump the img version of the runtime-watcher and since the coverage of tests for that webhook is not so good, it would be easy to push bugs and have a faulty img

Reply:
when the runtime-watcher is more stable, has more tests and is not changed frequently anymore, we can consider setting it to latest again :)